### PR TITLE
Expose IP Configurations for Microsoft.Network\VpnGateways

### PR DESCRIPTION
**Pull request by henry416**
## Description

Expose IP Configurations for VpnGateways as a read-only field in Get operations
Requires Network June nuget package (2020-06-01) to be merged to repo first.

Cmdlet review approval: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/621

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by henry416** [comment]
@VeryEarly SDK generation and addition to repo is usually done by another team, can you confirm are there any other items pending on this PR other than SDK generation?
This is a read-only value in Get Operation.

**Comment by VeryEarly** [comment]
> @VeryEarly SDK generation and addition to repo is usually done by another team, can you confirm are there any other items pending on this PR other than SDK generation?
> This is a read-only value in Get Operation.

Changes looks good to me,
In addition please add update changelog.md

**Comment by henry416** [comment]
@VeryEarly Done.

**Comment by VeryEarly** [comment]
> @VeryEarly Done.

LGTM, please upgrade SDK once it was published

**Comment by isra-fel** [comment]
Kindly ping @henry416

**Comment by henry416** [comment]
@isra-fel It seems like network package for this release still hasn't been uploaded, do you know who to contact? https://www.nuget.org/packages/Microsoft.Azure.Management.Network/

**Comment by isra-fel** [comment]
@MikhailTryakhov may be able to help :)

**Comment by MikhailTryakhov** [comment]
@henry416 you need to use our documentation (which exists for 2 years already and 2 years I'm asking you to READ it) 
please use local sdk. oficial build would be around Tuesday-Thursday next week

**Comment by henry416** [comment]
> @henry416 you need to use our documentation (which exists for 2 years already and 2 years I'm asking you to READ it)
> please use local sdk. oficial build would be around Tuesday-Thursday next week

Yes I am using local SDK, but I can't check in with a local SDK, we need the official nuget to be checked in. All testing using local has been done.

**Comment by henry416** [comment]
@MikhailTryakhov / @isra-fel

File: src/Network/Network/Models/Cortex/PSVpnGateway.cs [hunk 1]
```diff
@@ -31,5 +31,8 @@ public class PSVpnGateway : PSTopLevelResource
 
         [Ps1Xml(Label = "Provisioning State", Target = ViewControl.Table)]
         public string ProvisioningState { get; set; }
+
+        [Ps1Xml(Label = "IP Configurations", Target = ViewControl.Table)]
```
Comment (isra-fel) [inline src/Network/Network/Models/Cortex/PSVpnGateway.cs:35]:
```suggestion
```
Not a good idea to have too many columns in a table view. Powershell terminal isn't wide usually. Besides this is a complex object.
(Removing `[Ps1Xml]` attribute only removes it from the table view. User can still get the property by `$gateway.IpConfigurations` so it's safe.

**Comment by henry416** [comment]
@isra-fel Suggestion taken, any other pending items?

**Review by isra-fel — APPROVED** [review]

**Pull request merged**